tapfreighter/chain_porter.go

```diff
@@ -670,7 +666,8 @@ func (p *ChainPorter) transferReceiverProof(pkg *sendPackage) error {
 			return nil
 		}
 		if err != nil {
-			return fmt.Errorf("error delivering proof: %w", err)
```

Comment:
Nice catch! Too bad this couldn't be caught by any static linter. Because GoLand didn't catch this, even though I think under the hood it also runs gopls...

Reply:
Agree - it just got turned on by default for gopls 0.15 series, idk if adding a gopls run to CI would make sense.